pkg/controller/dbsyncer/subscription_db_to_transport_syncer.go

```diff
@@ -0,0 +1,38 @@
+package dbsyncer
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/bundle"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/db"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/intervalpolicy"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/transport"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	subscriptionsv1 "open-cluster-management.io/multicloud-operators-subscription/pkg/apis/apps/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	subscriptionsTableName = "Subscriptions"
```

Comment:
table name is lower case.. "subscriptions".
https://github.com/stolostron/hub-of-hubs-postgresql/blob/main/roles/create_tables/tasks/create_spec_tables.yaml#L12